framework/Models/PostProcessor.py

```diff
@@ -203,12 +203,13 @@ def submit(self,myInput,samplerType,jobHandler,**kwargs):
            a mandatory key is the sampledVars'that contains a dictionary {'name variable':value}
         @ Out, None
     """
-
+    kwargs['forceThreads'] = True
+    Model.submit(self,myInput, samplerType, jobHandler,**kwargs) 
     ## This may look a little weird, but due to how the parallel python library
     ## works, we are unable to pass a member function as a job because the
     ## pp library loses track of what self is, so instead we call it from the
     ## class and pass self in as the first parameter
-    jobHandler.addJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, str(0), modulesToImport=self.mods, forceUseThreads=True)
+    #jobHandler.addJob((self, myInput, samplerType, kwargs), self.__class__.evaluateSample, str(0), modulesToImport=self.mods, forceUseThreads=True)
```

Comment:
Same as above, justification for keeping commented code.

Reply:
removed